### DISCUSSION TITLE
Public \ corporate access policy

### DISCUSSION THREAD

**Discussion by NickA01** [co-cddo/api-catalogue#739] [Q&A]
Hi, 
I work for a private company ( non government.)
I've come across  the gov.uk/bank-holidays.json api and am wondering if it is acceptable for use outside of government organisations. 

I've had a look through the licensing agreement, ReadMe and a few of the pages on the cddo website and I cannot see anything in relation to usage by non goverment orgs. 
Please advise.

**Comment by stevenjmesser** [comment]
You must follow the terms of the MIT licence and Crown copyright. See here: https://github.com/alphagov/calendars/blob/master/LICENCE.txt
